linode/helper/config.go

```diff
@@ -18,7 +18,7 @@ import (
 const uaEnvVar = "TF_APPEND_USER_AGENT"
 
 // DefaultLinodeURL is the Linode APIv4 URL to use.
-const DefaultLinodeURL = "https://api.linode.com/v4"
+const DefaultLinodeURL = "https://api.linode.com"
```

Comment:
This is in response to the `SetAPIVersion(...)` call being corrected in `linodego v1.2.0`.